src/test/resources/features/completion/rules_and_checkpoints_names_completion.feature

```diff
@@ -392,9 +392,17 @@ Feature: Rule and Checkpoints names completion after 'rules.' and 'checkpoints.'
     Given a snakemake project
     And a file "boo.smk" with text
     """
+    include: "zoo.smk"
```

Comment:
It is better when a test checks a single idea/feature. Here we test several things together - completion from module file and from included file, from undefined module and local rule as other rule. Could you please split test in several independent tests that better reflect which case is being tested.